src/mscorlib/src/System/Text/ASCIIEncoding.cs

```diff
@@ -78,12 +78,22 @@ public override unsafe int GetByteCount(char* chars, int count)
             return EncodingForwarder.GetByteCount(this, chars, count);
         }
 
-        public override int GetBytes(String chars, int charIndex, int charCount,
-                                              byte[] bytes, int byteIndex)
-        {
-            return EncodingForwarder.GetBytes(this, chars, charIndex, charCount, bytes, byteIndex);
+#if !BIGENDIAN
+        public override byte[] GetBytes(String s)
+            => EncodingForwarder.GetBytesAsciiFastPath(this, s);
+
+        public override int GetBytes(String s, int charIndex, int charCount, byte[] bytes, int byteIndex)
```

Comment:
>String s [](start = 37, length = 8)

please keep the original paramter names . like in this case it should be "String chars"